lib/davinci_dtr_test_kit/payer_server_groups/payer_server_next_response_validation_test.rb

```diff
@@ -17,6 +17,7 @@ class PayerAdaptiveFormNextResponseTest < Inferno::Test
       This test may process multiple resources, labeling messages with the corresponding tested resources
       in the order that they were received.
     )
+    verifies_requirements 'hl7.fhir.us.davinci-dtr_2.0.1@169'
```

Comment:
requirement 169 isn't checked here - this requirement is specific to $questionnaire-package, but this tests $next-question

Reply:
Addressed in [6653e49](https://github.com/inferno-framework/davinci-dtr-test-kit/pull/59/commits/6653e495e069efcc7e8474e6becffb5493393339)